index.js

```diff
@@ -666,11 +666,13 @@ class WebUntis {
 
     /**
      * Get all classes known by WebUntis
+     * @param {number} schoolyearId
      * @param {boolean} [validateSession=true]
      * @returns {Promise.<Array>}
      */
-    async getClasses(validateSession = true) {
-        return await this._request('getKlassen', {}, validateSession);
+    async getClasses(schoolyearId = undefined,validateSession = true) {
```

Comment:
To maintain compatibility, please move `schoolyearId` behind the `validateSession` argument. And remove the default `undefined`, because it isn't needed then.